src/database/models/student-topic-assessment-override.ts

```diff
@@ -0,0 +1,110 @@
+import { Model, DataTypes } from 'sequelize';
+import appSequelize from '../app-sequelize';
+
+export interface StudentTopicAssessmentOverrideInterface {
+    id: number;
+    topicAssessmentInfoId: number;
+    userId: number;
+    duration: number | null;
+    maxGradedAttemptsPerVersion: number | null;
+    maxVersions: number | null;
+    versionDelay: number | null;
+    active: boolean;
+}
+
+export default class StudentTopicAssessmentOverride extends Model implements StudentTopicAssessmentOverrideInterface {
+    public id!: number;
+    // public courseTopicContentId!: number;
+    public topicAssessmentInfoId!: number;
+    public userId!: number;
+    public duration!: number | null;
```

Comment:
@red-tt-bear should these use `?` instead of `!`?

Reply:
Nah, this is from sequelize [documentation](https://sequelize.org/master/manual/typescript.html)

Reply:
Sorry I missed what you were saying, yeah I think the duration should have a question mark

Reply:
Actually, according to that documentation, this is correct
> public preferredName!: string | null; // for nullable fields

Reply:
I think we should leave it for now and deal with it later as it matches the documentation, I feel like there is a non 0 chance the documentation is wrong